.circleci/config.yml

```diff
@@ -369,6 +369,19 @@ jobs:
             - httplib.results
       - *save_cache_step
 
+  molten:
+    docker:
+      - *test_runner
+    steps:
+      - checkout
+      - *restore_cache_step
+      - run: tox -e 'molten_contrib-{py36}-molten{05,06,07}' --result-json /tmp/molten.results
```

Comment:
There is a fun conversation here around how far back do we want to support, especially since `0.1.0` looks like it is even from this year (July).

Reply:
Right, we might want to revisit this after we talk to a customer instrumenting molten. I'd be open to making it just 0.7 as the version we support out of the gate.